### PR TITLE
Add a sample playbook file

### PR DESCRIPTION
**Pull request by settermjd**
[Antora](https://docs.antora.org/antora/)'s still a bit young, so it doesn't - **yet** - provide documentation in all forms necessary to answer the questions that every user could have (okay, no project does a perfectly comprehensive job, but I hope you get what I'm saying here). 

So this commit adds a sample [Playbook file](https://docs.antora.org/antora/1.0/playbook/#playbook-file-formats), one that is heavily commented, so that contributors can better understand what can be configured, what the sections do, and where to find more information.

**Note:** It is also usable when building the ownCloud docs in what Antora refers to as [Author Mode](https://docs.antora.org/antora/1.0/playbook/author-mode/).

### CODE REVIEW

File: site.local.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+#
+# In this section, you can configure the UI/theme for the documentation. This
+# covers the UI theme bundle to use, and whether to use a default page layout.
+# For more information, see
+# https://docs.antora.org/antora/1.0/playbook/configure-ui/
+#
+ui:
+  bundle:
+    url: https://github.com/owncloud/docs-ui/releases/download/1.0.0/ui-bundle.zip
```
Comment (mmattel) [inline site.local.yml]: Why do we need ``releases`` instead using the directory ``bundle`` which exists and would be easy to un-gitignore (...gulp pack)? I may be wrong, but this approach implies a manual steep creating a release by someone after we have made a PR, commit, approval, merge process. Why a second process ?

File: site.local.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  output_dir: assets
+
```
Comment (mmattel) [inline site.local.yml]: If that gets included, I question myself, and that may be out of that PR (or not) which components (directory or files) are mandatory needed in the docs repro and which once are or can be included by the .yml file.
I am thinking of the docs/antora-ui-default/ directory. Tese things seem to be double present?

**Comment by mmattel** [comment]
when finished, will we add that stuff to our working site.yml ?

File: site.local.yml (hunk 1 above)
Comment (settermjd) [reply site.local.yml]: It was where I was at at the time that I originally created the file. @tboerger's implementing a build pipeline for the docs. I'll change this after he's finished.

File: site.local.yml (hunk 2 above)
Comment (settermjd) [reply site.local.yml]: I don't follow, sorry.

File: site.local.yml (hunk 1 above)
Comment (mmattel) [reply site.local.yml]: Let us not forget to update this accordingly :smile:

**Comment by mmattel** [comment]
``So this commit adds a sample Playbook file``...
Q: Why naming it ``site.local.ym`` and not ``site.sample.yml`` ?

File: site.local.yml (hunk 2 above)
Comment (mmattel) [reply site.local.yml]: Note: the directory ``docs/antora-ui-default/`` has been removed in the meanwhile by PR https://github.com/owncloud/docs/pull/209

**Comment by settermjd** [comment]
@mmattel, the file's been renamed to `site.sample.yml`. I compared it to `site.yml` and don't see any differences worth migrating, except for the attributes, potentially. However, custom attributes aren't used, yet, in the docs. However, it might be handy to add the section, even if it's empty, as a starting point.

File: site.local.yml (hunk 2 above)
Comment (settermjd) [reply site.local.yml]: I'm planning to remove bundle.url for the production playbook file, as it should be specified on the command-line by the build tool as part of the build pipeline.

**Pull request merged**